Sources/NIO/ChannelPipeline.swift

```diff
@@ -1555,6 +1555,20 @@ extension ChannelPipeline: CustomDebugStringConvertible {
         return desc.joined(separator: "\n")
     }
     
+    /// Returns the `ChannelHandlerContext` that belongs to a `ChannelHandler` of the given type.
```

Comment:
that documentation isn't quite right. It returns the first `ChannelHandler` of a given type.